src/stache/includes/cli/logLevel.html

```diff
@@ -0,0 +1 @@
+<stache-code>--logLevel</stache-code> — Specifies the logging information to provide in the command prompt. By default, this flag is set to <stache-code>--logLevel info</stache-code> to display all errors, warnings, and information. To only display errors, use <stache-code>--logLevel error</stache-code>. To only display errors and warnings, use <stache-code>--logLevel warn</stache-code>. To display all details for errors, warnings, and information, use <stache-code>--logLevel verbose</stache-code>.
```

Comment:
I think we need to tweak this one slightly.  `--logLevel verbose` will display all errors, warnings, informational, and verbose logs (like a level below `info`).